Workbooks/WorkbooksMetadata.json

```diff
@@ -657,7 +657,7 @@
     "dataTypesDependencies": [ "SecurityAlert", "ProtectionStatus", "SecurityRecommendation", "SecurityBaseline", "SecurityBaselineSummary", "Update", "ConfigurationChange" ],
     "dataConnectorsDependencies": [ "AzureSecurityCenter" ],
     "previewImagesFileNames": [ "ASCCaPBlack.png", "ASCCaPWhite.png" ],
-    "version": "1.1",
+    "version": "1.2",
```

Comment:
Why did you update this version I can't see changes in the Workspace audit workbook json itself?
the same for the provider field

Reply:
i was trying to fix the conflict that was created by other accepted PRs.  I have reverted this commit